__tests__/app.test.js

```diff
@@ -26,20 +26,24 @@ describe("/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then((response) => {
-        console.log(response.body.topics);
         expect(response.body.topics.length).toBe(3);
         response.body.topics.forEach((topic) => {
           expect(topic).toHaveProperty("slug", expect.any(String));
           expect(topic).toHaveProperty("description", expect.any(String));
         });
       });
   });
-  test("POST: should return 405 for invalid method", () => {
+});
+
+describe("/api", () => {
+  test("GET:200 sends documentation for available endpoints to the client", () => {
     return request(app)
-      .post("/api/topics")
-      .expect(405)
+      .get("/api")
+      .expect(200)
       .then((response) => {
-        expect(response.body.msg).toBe("Method not allowed");
+        expect(response.body.docs).toHaveProperty("GET /api");
+        expect(response.body.docs).toHaveProperty("GET /api/topics");
+        expect(response.body.docs).toHaveProperty("GET /api/articles");
```

Comment:
This test will break on the next endpoint and need to be updated which makes it very brittle and goes against the TDD principles we have taught. What could you do to make this test more dynamic? The json file can be required in as a json object you can interact with.